spec/system/admin/reports_spec.rb

```diff
@@ -560,12 +560,19 @@
 
       it "can customise a number of fields" do
         fill_in 'initial_invoice_number', with: '5'
-        fill_in 'invoice_date', with: '2015-02-12'
-        fill_in 'due_date', with: '2015-03-12'
+        find('#invoice_date').click
+        select_datetime_from_datepicker Time.zone.at(Time.zone.local(2021, 02, 12))
+        # hide the datetimepicker
+        find("body").send_keys(:escape)
```

Comment:
Shall we introduce a helper for this? Maybe used like this:
```rb
pick_date "#invoice_date", Date.new(2021, 2, 12)

# And / or:
pick_datetime "#orders_created_at_from", DateTime.new(2021, 2, 12, 10)
```

I don't think that we have a time zone in the datepicker to consider.